### PR TITLE
docs(localize): fix typo in i18n-overview.md

### PR DESCRIPTION
**Pull request by PzYon**

### CODE REVIEW

**Comment by AndrewKushnir** [comment]
@PzYon thanks for creating the PR. Could you please update the commit message (`git commit --amend`) to conform [the commit message guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)? You have the correct PR title, but the underlying commit is missing the `docs(localize):` part.

File: aio/content/guide/i18n-overview.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ A locale determines the formatting and parsing of the following details.
     </a>
     <a href="guide/i18n-example" class="docs-card" title="Internationalization example">
         <section>Internationalization example</section>
-        <p>Review an example fo Angular internationalization.</p>
+        <p>Review an example for Angular internationalization.</p>
```
Comment (AndrewKushnir) [inline aio/content/guide/i18n-overview.md:32]: Should that be `of` instead? Looks like this was a typo of `of`.
```suggestion
        <p>Review an example of Angular internationalization.</p>
```

**Comment by PzYon** [comment]
Hey @AndrewKushnir , thanks for your feedback. I'm not a native English speaker, but I guess `of` is better. ;) Commit message is updated too. Thanks for your support.

**Review by AndrewKushnir — APPROVED** [review]
@PzYon thanks for updating the PR! The change looks good and I'm adding this PR to the merge queue.

Thanks for improving Angular docs! 👍

**Comment by AndrewKushnir** [comment]
@PzYon I just noticed that there are 2 commits for some reasons. Could you please merge both commits into one and let us know (by leaving a comment here) once it's done, so I can add the PR back to the merge queue?

**Comment by PzYon** [comment]
Done. :)

**Comment by AndrewKushnir** [comment]
@PzYon thanks for the update! I'm adding this PR back to the merge queue.

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 80ab604fe00eee18083cb0840577804b3fd9bc7b.

**Pull request closed without merging**